### DISCUSSION TITLE
Can not change LDAP user's role

### DISCUSSION THREAD

**Discussion by assilzm** [kanboard/kanboard#5203] [Installation & Configuration]
### Actual behaviour

I changed a LDAP user's role, nomal user to admin.When I relogin this user,it's role reset to nomal user.

### Expected behaviour

I can change the LDAP user's role currectly.

### Configuration

- Kanboard version: 1.2.20.Docker version.
- Database type and version:
- PHP version: 7.4.19
- OS: Centos 7.4
- Browser: Chrome
- No plugins

just like #2198

**Comment by fguillot** [comment]
Try to disable the synchronization of the role attribute.

```
// This example will not synchronize the field "role" from LDAP to Kanboard.
define('EXTERNAL_AUTH_EXCLUDE_FIELDS', 'role');
```

https://docs.kanboard.org/v1/admin/ldap/#synchronization
